src/components/shared/logo.tsx

```diff
@@ -9,37 +9,51 @@ import { Item, getFormat } from 'item';
 import { pipe2 } from 'lib';
 import { map, withDefault } from '@guardian/types/option';
 import { textSans } from '@guardian/src-foundations/typography';
+import { getPillarStyles } from 'pillarStyles';
 
 interface Props {
     branding: Branding;
     format: Format;
 }
 
-const styles = (lightModeImage: string, darkModeImage?: string): SerializedStyles => css`
-    margin: ${remSpace[9]} 0;
-    ${textSans.small()}
+const styles = (format: Format, 
+                lightModeImage: string, 
+                darkModeImage?: string): SerializedStyles => {
+    const { kicker, inverted } = getPillarStyles(format.pillar);
+    return css`
+        margin: ${remSpace[9]} 0;
+        ${textSans.small()}
 
-    img {
-        content: url("${lightModeImage}");
-        display: block;
-        margin: ${remSpace[2]} 0;
-        max-height: 60px;
-    }
-
-    label {
-        color: ${text.supporting};
-    }
-
-    ${darkModeCss`
         img {
             content: url("${darkModeImage ?? lightModeImage}");
```

Comment:
I believe this is the image URL in light mode?

```suggestion
            content: url("${lightModeImage}");
```

Reply:
Not 100% sure to be honest, I assumed `darkModeImage ??` meant it would load a different image in dark mode, and if one was not found it would back up to the original dark mode.

Reply:
`??` is the [nullish coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator) - it attempts to use the first value, but if that value is `null` or `undefined` it falls back to the second. In this case it attempts to use the value of `darkModeImage`, and if that value is missing it falls back to using the `lightModeImage`. This is desirable in situations where we've only been provided with a light mode version of a logo: the code will use that for both light mode and dark mode.

However, we only want to make an attempt to use a dark mode variant of the logo *when in dark mode*. Our `darkModeCss` function is what allows us to set styles that are only applied when the user is in dark mode. Therefore, any CSS relevant to dark mode should *only be applied via this function*. In summary, on this line we're not using the `darkModeCss` function, so we only want to provide the `lightModeImage`. Down on line 44 we *are* using the `darkModeCss` function so that's where we provide the `darkModeImage`.

Reply:
Ah ok, thanks for that, to be honest i was looking at the dark mode css, so it is redundant where it is as we are not in dark mode at that part of the css.
Thanks for the explanation by the way I do really appreciate the time detail these help me a great deal (I copy them into my notes for later reference when stuck!)
Also i was looking at the line 44 instance of it 🤦‍♂️